packages/cross_file/lib/src/types/html.dart

```diff
@@ -114,10 +117,10 @@ class XFile extends XFileBase {
 
     // Create an <a> tag with the appropriate download attributes and click it
     // May be overridden with CrossFileTestOverrides
-    final AnchorElement element =
-        (_hasTestOverrides && _overrides.createAnchorElement != null)
-            ? _overrides.createAnchorElement(this.path, this.name)
-            : createAnchorElement(this.path, this.name);
+    final AnchorElement element = _hasTestOverrides
+        ? _overrides!.createAnchorElement(this.path!, this.name ?? '')
```

Comment:
The path is non-null, because it's either mandatory in the normal constructor, or it gets artificially constructed in the "fromData" constructor. I might be able to mark it "late" instead of "?"

Reply:
(Ended up marked as `late String`)